src/orchestrator.ts

```diff
@@ -33,11 +33,11 @@ export const orchestrator = async function () {
   );
   const scrapers: Scraper[] = activeMarkets.map((market) => new Scraper(market, uploader));
 
-  // we need to use a cron lock concept because the scrapers are stateful anyway
-  let cronLocked = false;
+  /* Use a cron lock concept because the scrapers are stateful */
+  let isCronLocked = false;
   cron.schedule(`*/${SECONDS_BETWEEN_RUNS} * * * * *`, async () => {
-    if (!cronLocked) {
-      cronLocked = true;
+    if (!isCronLocked) {
```

Comment:
ive found that naming all booleans as questions to be super useful & clarifying when reading later